### ISSUE TITLE
Option to provide notifications for images that haven't been updated in X days

### ISSUE THREAD

**Issue by zacps** [containrrr/watchtower#1772]
### Is your feature request related to a problem? Please describe.

I run some images for applications from random people in the community. Some of these images are no longer maintained.

This can have security implications if they haven't received necessary updates.

### Describe the solution you'd like

I would like to be able to configure watchtower to send a notification if an image has not been updated in X days, for example I would probably set this to about a year. This way I can identify images which are no longer maintained and swap them out for alternatives.

### Describe alternatives you've considered

This could be a standalone application, but watchtower already checks for image updates and can send notifications, so it seems a sensible place to do it.

**Comment by simonaronsson** [comment]
This sounds like a really nifty feature to have. Would you be interested in having a stab at it? I'm happy to assist with code reviews and such.

**Comment by zacps** [comment]
Absolutely! I'm planning on setting some time aside for open source contributions for Hacktoberfest, so I'll try to get to it then :).
